sender-stackdriver/src/main/java/zipkin2/reporter/stackdriver/StackdriverSender.java

```diff
@@ -164,10 +164,39 @@ public void close() {
     ((ManagedChannel) channel).shutdownNow();
   }
 
-  final class PatchTracesCall extends UnaryClientCall<PatchTracesRequest, Empty> {
+  Span parseTraceIdPrefixedSpan(byte[] traceIdPrefixedSpan) {
```

Comment:
plug.. if anyone gets to the point of groking this or upvoting, we can actually guarantee most span data comes in the same hunk (per trace ID) https://github.com/openzipkin/brave/pull/801